charter/index.html

```diff
@@ -168,19 +169,21 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
         <h2>Motivation and Background</h2>
         <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created it.</p>
 
-        <p>Solid extends open web standards to give priority to individual and community autonomy, allowing them to keep authority over their identity, data, and privacy, and giving them the freedom to select applications and services that suit their needs.</p>
+        <p>Solid is largely based on existing open web standards, and extends them to give priority to individual and community autonomy. It allows them to keep authority over their identity, data, and privacy, and gives them the freedom to select applications and services that suit their needs.</p>
```

Comment:
The grammatical structure of "It" refers back to the subject "Solid" from the previous sentence and not "individual and community", as it seems to be what's intended. Perhaps you meant something along these lines:

```suggestion
        <p>Solid is largely based on existing open web standards, and extends them to give priority to individual and community autonomy. Solid allows both individuals and communities to keep authority over their identity, data, and privacy, and gives them the freedom to select applications and services that suit their needs.</p>
```

Reply:
> The grammatical structure of "It" refers back to the subject "Solid" from the previous sentence 

yes

> and not "individual and community", as it seems to be what's intended.

No, it was indeed intended to refer to Solid.

> Perhaps you meant something along these lines: (...)

Yes, that's what I meant. We seem to agree that "It" grammatically refers to "Solid", and "them" refers to "individuals and communities". Where is the problem then??

I don't mind adopting your proposed changed, but I find it more verbose and slightly redundant.